client/tests/test_serving.py

```diff
@@ -0,0 +1,181 @@
+import csv
+import shutil
+import time
+from tempfile import NamedTemporaryFile
+from unittest import TestCase
+import os, stat
+import pandas as pd
+import pytest
+from featureform import ServingClient, ResourceClient
+
+from localmode_cases import features_no_ts, features_with_ts, feature_invalid_entity, feature_invalid_value, \
```

Comment:
[conftest](https://www.tutorialspoint.com/pytest/pytest_conftest_py.htm) might be a better approach getting the features.

Reply:
Is there a better approach than adding them all into a single conftest file? The localmode file is getting pretty long and this is just the first set of cases

Reply:
the conftest would allow the fixtures to be used across different test files without importing it. If they are only being used in a particular file, we can add them as fixtures to that file.